### ISSUE TITLE
label/dict.txt for CDS classifier

### ISSUE THREAD

**Issue by mrzhang11** [martiansideofthemoon/style-transfer-paraphrase#5]
Hi,  could you provide the CDS-data-bin/label/dict.txt that you used during fine-tuning to map label_ids and labels? 
It will be much helpful for running the pre-trained CDS classifier.

Thanks very much!

**Comment by martiansideofthemoon** [comment]
Hey sure, sorry for the omission and the delay with all this. Here's the file: https://drive.google.com/drive/u/1/folders/1b0YkZipzcrI9lZvJJ2_FibGFKA-wwYFA
